staging/src/k8s.io/apiserver/pkg/util/webhook/webhook.go

```diff
@@ -32,13 +32,21 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
+// defaultRequestTimeout is set for all webhook request. This is the absolute
+// timeout of the HTTP request, including reading the response body.
+const defaultRequestTimeout = 20 * time.Second
+
 type GenericWebhook struct {
 	RestClient     *rest.RESTClient
 	initialBackoff time.Duration
 }
 
 // NewGenericWebhook creates a new GenericWebhook from the provided kubeconfig file.
 func NewGenericWebhook(registry *registered.APIRegistrationManager, codecFactory serializer.CodecFactory, kubeConfigFile string, groupVersions []schema.GroupVersion, initialBackoff time.Duration) (*GenericWebhook, error) {
+	return newGenericWebhook(registry, codecFactory, kubeConfigFile, groupVersions, initialBackoff, defaultRequestTimeout)
```

Comment:
Should make this configurable?

Reply:
Configurable by who? You mean adding new API flags or adding this value to the admission webhook API?

Reply:
I am not sure ,  may be this 
>// TODO(ericchiang): Make these value configurable. Maybe through a
>	// kubeconfig extension?

Reply:
If we're not sure let's just leave it as is. If there's a strong reason to control this timeout we can argue for that later.